Authorization/src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -0,0 +1,100 @@
+<!--        </preConditions>-->
+<!--        <createTable tableName="users_roles">-->
+<!--            <column name="role_id" type="int"/>-->
+<!--            <column name="user_id" type="int"/>-->
+<!--        </createTable>-->
+<!--    </changeSet>-->
+
+<!--&lt;!&ndash;        Create the table role if does not exist&ndash;&gt;-->
+<!--    <changeSet id="2" author="Nikolay2">-->
+<!--        <preConditions onFail="CONTINUE">-->
+<!--            <not>-->
+<!--                <tableExists tableName="role"/>-->
+<!--            </not>-->
+<!--        </preConditions>-->
+<!--        <createTable tableName="role">-->
+<!--            <column name="id" type="int">-->
+<!--                <constraints primaryKey="true" uniqueConstraintName="role_id_unq"/>-->
+<!--            </column>-->
+<!--            <column name="name" type="varchar(250)">-->
+<!--                <constraints unique="true" uniqueConstraintName="role_name_unq"/>-->
+<!--            </column>-->
+<!--        </createTable>-->
+<!--        <addForeignKeyConstraint baseTableName="role" baseColumnNames="id" constraintName="role_id_fk"-->
+<!--                                 referencedTableName="users_roles" referencedColumnNames="role_id"/>-->
+<!--    </changeSet>-->
+
+<!--    &lt;!&ndash;    Create the table user if does not exist&ndash;&gt;-->
+<!--    <changeSet id="3" author="Nikolay3">-->
+<!--        <preConditions onFail="CONTINUE">-->
+<!--            <not>-->
+<!--                <tableExists tableName="user"/>-->
+<!--            </not>-->
+<!--        </preConditions>-->
+<!--        <createTable tableName="user">-->
+<!--            <column name="id" type="int">-->
+<!--                <constraints primaryKey="true" primaryKeyName="id"/>-->
+<!--            </column>-->
+<!--            <column name="username" type="varchar(250)">-->
+<!--                <constraints unique="true" uniqueConstraintName="username_unq"/>-->
+<!--            </column>-->
+<!--            <column name="password" type="varchar(250)"/>-->
+<!--            <column name="email" type="varchar(250)">-->
+<!--                <constraints unique="true" uniqueConstraintName="email_unq"/>-->
+<!--            </column>-->
+<!--            <column name="refresh_token" type="varchar(250)">-->
+<!--                <constraints unique="true" uniqueConstraintName="refresh_token_unq"/>-->
+<!--            </column>-->
+<!--        </createTable>-->
+<!--        <addAutoIncrement tableName="user" columnName="id" columnDataType="int"/>-->
+<!--        <addForeignKeyConstraint baseTableName="user" baseColumnNames="id" constraintName="user_id_fk"-->
+<!--                                 referencedTableName="users_roles" referencedColumnNames="user_id"/>-->
+
+        <!--        If the table user is empty, then inserts admin and this admin will be added in users_roles table -->
+<!--    </changeSet>-->
+    <changeSet id="4" author="Nikolay4">
+        <preConditions onFail="CONTINUE">
```

Comment:
Not sure `onFail=CONTINUE` is adequate for such an important migration.
If liquibase simply continues after failing to create the users table, then the app will not work.
It's better to fail and let the app crash entirely, so that health checks for example, can give alerts that the app is down / not working.